students/piechowski_michal/lesson_06_dicts_tuples_sets_args_kwargs/logger_wrapper.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+
+
+def foo(*args, **kwargs):
+    print("\nfoo called with:")
+    print(args, kwargs)
+
+
+def logger_wrapper(*args, **kwargs):
```

Comment:
To chyba nie do końca tak miało być.
`foo` jest parametrem do funkcji `logger_wrapper` i prezentuje nazwę dowolnej funkcji, która jest wywoływań przez funkcje 'logger_wrapper`:
`logger_wrapper(jakas_nazwa_funkcji, param1, param2, param3, kparm1=1, kparm2=2, kparm3=3)`
przy wywołaniu, tego funkcja wyświetli informacje na temat arg i kwargs i wywoła podaną funkcje.
Możesz tutaj wykorzystać funkcję `args inspector` z poprzedniego zadania.